pkg/proxy/proxy.go

```diff
@@ -155,33 +156,41 @@ func (p *Proxy) health(ctx echo.Context) error {
 }
 
 func (p *Proxy) handleRequestAndRedirect(ctx echo.Context) error {
+	startTime := time.Now()
```

Comment:
Maybe it's better to create another middleware to put the start time into the context? And add this midleware as the first one here: https://github.com/codeready-toolchain/registration-service/blob/d34df63d6652c49c8739b14cd78f477441c6472f/pkg/proxy/proxy.go#L94
Then all the routes will have the start time available in the context OOTB.

Reply:
@alexeykazakov done in 79447b8